manifest.yml

```diff
@@ -7,6 +7,4 @@ github:
 binary: example
 commands: 
     testplugin: 
```

Comment:
```suggestion
    example: 
```

Reply:
I think this is the name of the command, right?

Reply:
yes I changed the wrong value by accident. Fixed it now